src/plugins/embeddable/public/lib/panel/panel_header/panel_header.tsx

```diff
@@ -62,16 +62,32 @@ function renderNotifications(
   notifications: Array<Action<EmbeddableContext>>,
   embeddable: IEmbeddable
 ) {
-  return notifications.map(notification => (
-    <EuiNotificationBadge
-      data-test-subj={`embeddablePanelNotification-${notification.id}`}
-      key={notification.id}
-      style={{ marginTop: '4px', marginRight: '4px' }}
-      onClick={() => notification.execute({ embeddable })}
-    >
-      {notification.getDisplayName({ embeddable })}
-    </EuiNotificationBadge>
-  ));
+  return notifications.map(notification => {
+    const context = { embeddable };
+
+    let badge = (
+      <EuiNotificationBadge
+        data-test-subj={`embeddablePanelNotification-${notification.id}`}
+        key={notification.id}
+        style={{ marginTop: '4px', marginRight: '4px' }}
+        onClick={() => notification.execute(context)}
+      >
+        {notification.getDisplayName(context)}
+      </EuiNotificationBadge>
+    );
+
+    const tooltip = notification.getDisplayNameTooltip!(context);
+
+    if (tooltip) {
+      badge = (
+        <EuiToolTip position="left" delay="regular" content={tooltip}>
```

Comment:
Let's change to position=top so we can avoid any potential overlay with the panel title.

Reply:
Done.

Reply:
![image](https://user-images.githubusercontent.com/9773803/81041432-21c16e80-8eae-11ea-91c7-2ba23fc39d23.png)